setup.py

```diff
@@ -1,53 +1,26 @@
 #!/usr/bin/env python
 import io
-from setuptools import setup, find_packages
-
-
-def is_requirement(line):
-    """
-    Return True if the requirement line is a package requirement;
-    that is, it is not blank, a comment, or editable.
-    """
-    # Remove whitespace at the start/end of the line
-    line = line.strip()
-
-    # Skip blank lines, comments, and editable installs
-    return not (
-        line == '' or
-        line.startswith('-r') or
-        line.startswith('#') or
-        line.startswith('-e') or
-        line.startswith('git+')
-    )
 
+from setuptools import setup, find_packages
 
-def load_requirements(*requirements_paths):
-    """
-    Load all requirements from the specified requirements files.
-    Returns a list of requirement strings.
-    """
-    requirements = set()
-    for path in requirements_paths:
-        requirements.update(
-            line.strip() for line in open(path).readlines()
-            if is_requirement(line)
-        )
-    return list(requirements)
-
+import release_util
 
-long_description = io.open('README.md', encoding='utf-8').read()
+long_description = io.open('README.rst', encoding='utf-8').read()
 
 METADATA = dict(
     name='edx-django-release-util',
-    version='0.0.5',
+    version=release_util.__version__,
```

Comment:
Will moving the version from 0.0.5 to 0.0.1 confuse PyPI publishing?

https://pypi.python.org/pypi/edx-django-release-util/0.0.5